src/hotspot/share/opto/parse2.cpp

```diff
@@ -3503,12 +3503,6 @@ void Parse::do_one_bytecode() {
   case Bytecodes::_new:
     do_new();
     break;
-  case Bytecodes::_aconst_init:
-    do_aconst_init();
-    break;
-  case Bytecodes::_withfield:
-    do_withfield();
```

Comment:
Could you add a `// TODO JDK-8325660` to both methods so we don't forget removing them?

Reply:
do_withfield() and do_aconst_init() methods have been removed from C2 parse* code.